code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -242,9 +242,9 @@
 	key_third_person = "hisses"
 	message = "hisses!"
 	emote_type = EMOTE_AUDIBLE
-	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
 	vary = TRUE
 	sound = 'sound/emotes/hiss.ogg'
+	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
```

Comment:
this isn't a typecache, this is a list.

Reply:
`mob_type_allowed_typecache`, `mob_type_blacklist_typecache`, and `mob_type_ignore_stat_typecache` all get made into typecaches in `/datum/emote/New()`:
https://github.com/pariahstation/Pariah-Station/blob/434e9eecaf65c0b0b0091b32585ff2054db7811d/code/datums/emotes.dm#L65-L75